addons/mail/models/__init__.py

```diff
@@ -20,6 +20,7 @@
 from . import mail_activity
 from . import mail_blacklist
 from . import mail_followers
+from . import mail_ice_server
```

Comment:
This should be common with what we do in https://github.com/odoo/odoo/pull/66611

I'm not sure what is the target for this PR, but please don't merge it before the one I linked which should happen today if all goes right.

Reply:
Sure, theses files are copy-pasted from that branch.
I already concacted @ThanhDodeurOdoo  about that.
We will wait for the PR #66611.